### ISSUE TITLE
failed build on master branch according to travis - linked from readme

### ISSUE THREAD

**Issue by matkoniecz** [atom/atom#11849]
### Prerequisites
- [x]  [not relevant] Can you reproduce the problem in [safe mode](http://flight-manual.atom.io/hacking-atom/sections/debugging/#check-if-the-problem-shows-up-in-safe-mode)?
- [x]  [not relevant] Are you running the [latest version of Atom](http://flight-manual.atom.io/hacking-atom/sections/debugging/#update-to-the-latest-version)?
- [x] Did you check the [debugging guide](http://flight-manual.atom.io/hacking-atom/sections/debugging/)?
- [x] Did you check the [FAQs on Discuss](https://discuss.atom.io/c/faq)?
- [x] Are you reporting to the [correct repository](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#atom-and-packages)?
- [x] Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3Aatom+-repo%3Aatom%2Felectron) to see if your bug or enhancement is already reported?

For more information on how to write a good [bug report](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-do-i-submit-a-good-bug-report) or [enhancement request](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-do-i-submit-a-good-enhancement-suggestion), see the `CONTRIBUTING` guide.
### Description

failed build on master branch according to travis - linked from readme
### Steps to Reproduce
1. open https://github.com/atom/atom
2. see rendering of README.md 
3. see red badges
4. follow to recent master test at https://travis-ci.org/atom/atom/branches (at time of writing fail was caused by https://travis-ci.org/atom/atom/jobs/132892776 ).

**Expected behavior:** test on master branch are working

**Actual behavior:** either test or tested code is broken. Maybe readme link ignored tests, in that case badges should be removed from readme.
### Versions

Not relevant.

**Comment by hultberg** [comment]
Tests on master is broken. when clicking the badge you see the latest test runned in the whole project, which might not be master branch. See https://travis-ci.org/atom/atom/branches

**Comment by matkoniecz** [comment]
Well, that is an interesting approach. I already encountered projects where reporting bugs is pointless as due to project size number of bugs is massive and probably nearly everything was reported already (it probably makes sense to report security issues).

But "let's ignore bug reports and close them because we ignored them" is a new one.

I should remember to never waste time reporting issues in this or other Github's sideprojects.
